index.php

```diff
@@ -7,39 +7,19 @@
  */
 
 /**
- * Define the application's minimum supported PHP version as a constant so it can be referenced within the application.
- */
-define('JOOMLA_MINIMUM_PHP', '5.3.10');
-
-if (version_compare(PHP_VERSION, JOOMLA_MINIMUM_PHP, '<'))
-{
-	die('Your host needs to use PHP ' . JOOMLA_MINIMUM_PHP . ' or higher to run this version of Joomla!');
-}
-
-/**
  * Constant that is checked in included files to prevent direct access.
- * define() is used in the installation folder rather than "const" to not error for PHP 5.2 and lower
+ * define() is used in the installation folder rather than "const" to
+ * not error for PHP 5.2 and lower
  */
 define('_JEXEC', 1);
 
-if (file_exists(__DIR__ . '/defines.php'))
```

Comment:
It looks like we remove the ability to override the app defines, is this intended?  Truthfully we aren't in a state to offer this, so I agree with it, probably best to get opinions from others before committing to it.